lib/prompts/checkbox.js

```diff
@@ -96,7 +96,8 @@ Prompt.prototype.render = function (error) {
     message += chalk.cyan( this.selection.join(", ") );
   } else {
     var choicesStr = renderChoices(this.opt.choices, this.pointer);
-    message += "\n" + this.paginator.paginate(choicesStr, this.pointer, this.opt.pageSize);
+    message += "\n" + this.paginator.paginate(choicesStr, this.pointer,
+     this.opt.pageSize, this.getSeparatorOffset(this.pointer));
```

Comment:
Also, I feel like the correct position really just is `this.opt.choices.choices.indexOf(activeChoice)`. As this will position the choice item in the full array (with separators).